client/src/pages/SignUp/SignUpForm/useStyles.ts

```diff
@@ -6,28 +6,28 @@ const useStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    fontSize: 15,
+    fontWeight: 700,
+    color: 'black',
+    marginBottom: '-0.8rem',
+    marginTop: '0.8rem',
   },
   inputs: {
-    marginTop: '.8rem',
-    height: '2rem',
-    padding: '5px',
+    height: '1.5rem',
   },
   forgot: {
     paddingRight: 10,
     color: '#3a8dff',
   },
   submit: {
-    margin: theme.spacing(3, 2, 2),
+    margin: theme.spacing(3, 2, -1),
     padding: 10,
     width: 160,
     height: 56,
     borderRadius: theme.shape.borderRadius,
     marginTop: 49,
     fontSize: 16,
-    backgroundColor: '#3a8dff',
+    backgroundColor: '#primary',
```

Comment:
what is #primary color here? is this a valid color? did you mean to use the theme's primary colour?